go/master/c/client.go

```diff
@@ -101,6 +100,12 @@ func paddle_release_master_client(client C.paddle_master_client) {
 	remove(client)
 }
 
+//export paddle_start_get_records
+func paddle_start_get_records(client C.paddle_master_client, pass C.int) {
```

Comment:
Say if the job trains on pass 100, and one trainer gets killed and started again, should it set local pass to 0 and start from there?